spark-3.5.yaml

```diff
@@ -224,6 +245,94 @@ subpackages:
           ln -s /usr/lib/spark/sbin ${{targets.subpkgdir}}/opt/bitnami/spark/sbin
           ln -s /usr/lib/spark/yarn ${{targets.subpkgdir}}/opt/bitnami/spark/yarn
 
+  - name: ${{package.name}}-scala-2.13
+    dependencies:
+      runtime:
+        - openjdk-17-default-jvm
+    pipeline:
+      - runs: |
+          export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
+          export MAVEN_OPTS="-Xss64m -Xmx2g -XX:ReservedCodeCacheSize=1g"
+          ## pom bump seem to mess with the ability to switch scala version
+          ## the solution is to restore the original pom file at the package build stage
+          cp pom.xml.orig pom.xml
+          mvn clean
+          ./dev/change-scala-version.sh 2.13
+          ./build/mvn -DskipTests -Pscala-2.13 clean package
+          mkdir -p ${{targets.contextdir}}/usr/lib/spark
+          mkdir -p ${{targets.contextdir}}/usr/lib/spark/work-dir
+          mv bin/ ${{targets.contextdir}}/usr/lib/spark
+          mv sbin/ ${{targets.contextdir}}/usr/lib/spark
+          mv target ${{targets.contextdir}}/usr/lib/spark
+          mv assembly ${{targets.contextdir}}/usr/lib/spark
+          cp resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh ${{targets.contextdir}}/usr/lib/spark/
+
+  - name: ${{package.name}}-scala-2.13-compat
```

Comment:
Why do we need another compat package?

Reply:
@EyeCantCU The `entrypoint.sh` is sourced from a different location for `2.12` which I suspect is as a result of the build process.

Reply:
If we loop over a range of supported configurations (scala 2.12-2.13, jdk8-17), we can accommodate that in the same subpackage definition without creating a new one with much of the same logic

Reply:
@EyeCantCU Ranging over two different data items here might be difficult or impossible from my understanding. Is nested ranging supported?

Reply:
Replaced this with a range and conditional build